docs/user/alerting/action-variables.asciidoc

```diff
@@ -5,405 +5,230 @@
-
-For example, if you have data like this available as Mustache variables:
-
-[source,json]
-----
-{
-  "date": "2023-04-27T22:40:34.153Z",
-  "context": {
-    "title": "rule 'esq' matched query for group host-2",
-    "conditions": "Number of matching documents for group \"host-2\" is less than 1000",
-    "link": "https://example.com/this-will-link-to-Discover",
-    "hits": [
-      {
-        "_source": {
-          "event": {
-            "provider": "alerting",
-            "action": "active-instance",
-            "duration": "96023000000"
-          }
-        }
-      },
-      {
-        "_source": {
-          "@timestamp": "2023-04-27T22:40:22.251Z",
-          "event": {
-            "provider": "alerting",
-            "action": "execute-action"
-          }
-        }
-      }
-    ]
-  }
-}
-----
-
-You can create the following Mustache template in the email action for your rule:
-
-[source,js]
-----
-# {{context.title}} <1>
-
-{{#FormatDate}} {{{date}}} ; America/New_York {{/FormatDate}} <2>
-
-{{context.conditions}}
-
-**documents** _[view in Discover]({{{context.link}}})_ <3>
-
-| provider | action | duration | <4>
-| -------- | ------ | -------- |
-{{#context.hits}}{{#_source.event}}| {{provider}} | {{action}} | {{#duration}}{{#EvalMath}} round( {{{duration}}} / 1000 / 1000 / 1000 ) {{/EvalMath}} sec{{/duration}} {{^duration}}-n/a-{{/duration}} |{{/_source.event}}
-{{/context.hits}}
-----
-<1> Renders the value of the `context.title` variable as a level 1 heading.
-<2> Renders the value of the `date` variable as a formatted date in the America/New_York time zone.
-<3> Shows examples of `**bold**`, `_italic_`, and `[text](url)` links.
-<4> Shows a table with three columns, with one row per element in the `context.hits` array.
-From each of those elements, you can access the `provider`, `action`, and `duration` fields of the `_source.event` object.
-The `duration` field is rendered as a number of seconds, rounded to the nearest second.
-It's stored as nanoseconds so it needs to be divided by a billion to yield seconds.
-The duration field is optional, so you can use a `{{#duration}} ... {{/duration}}` section to render the duration if it's present and show `-n/a-` otherwise.
-
-When rendered into Markdown and then HTML and viewed in an email client, it looks like this:
-
-[role="screenshot"]
-image:images/email-mustache-template-rendered.png[Email template rendered in an email client]
+[[alert-actiongroup-variable]]`alert.actionGroup`::
+(string) The ID of the action group of the alert that scheduled the action. For example, `query matched`.
+[[alert-actiongroupname-variable]]`alert.actionGroupName`::
+(string) The name of the action group of the alert that scheduled the action. For example, `Query matched`.
+[[alert-actionsubgroup]]`alert.actionSubgroup`::
+(string) The action subgroup of the alert that scheduled the action.
+//TBD When does the subgroup appear? Is it related to the subcategory of ServiceNow connectors?
```

Comment:
You can remove references to `actionSubgroup` - we had it for a while, but then removed it.  The doc was never fixed too remove it.